common/paths.yaml

```
@@ -172,7 +172,7 @@
 /v2/sys/free:
   get:
     $ref: ../firmware/paths.yaml#/FreeGet
-/v2/sys/backup:
+/v2/sys/snapshot:
```

Looks like it was implemented as `backup` actually
```suggestion
/v2/sys/backup:
```